src/puppetlabs/trapperkeeper/core.clj

```diff
@@ -173,10 +173,12 @@
           (internal/parse-cli-args!)
           (run))
       (catch map? m
-        (case (without-ns (:type m))
-          :cli-error (quit 1 (:message m) *err*)
-          :cli-help (quit 0 (:message m) *out*)
-          (throw+)))
+        (let [type (:type m)]
+          (if (keyword? type)
```

Comment:
Is it at all possible there might be a `:type` keyword that's not `cli-error` or `cli-help` and we might want to handle that appropriately?  Idk if that would actually happen/what the whole path to get here is, but it seems like maybe having an `:else` for that case might be reasonable?